asmcomp/riscv/proc.ml

```diff
@@ -55,7 +56,7 @@ let word_addressed = false
   Additional notes
   ----------------
 
-    - t0-t1 are used by the assembler and code generator, so
+    - t1 is used by the assembler and code generator, so
```

Comment:
Is it really "used by the assembler"?  The MIPS and Alpha assemblers silently used a temporary register for some macro-instructions.  I don't think this is the case for the RISC-V assembler.

Reply:
Indeed you are right. the original comment meant that it was used in `riscv.S`, but it is confusing. I amended the comment accordingly.